### DISCUSSION TITLE
Is possible create Docker image for acelerate?

### DISCUSSION THREAD

**Discussion by guissalustiano** [AccelerateHS/accelerate#487] [Q&A] [answered]
I am having trouble installing accelerate and I believe it would be easy to create an image for anyone facing the same problem.
It doesn't seem too difficult to create a docker image

**Comment by tomsmeding** [comment]
What trouble are you having with the installation?

For the GPU functionality, we'd need to use nvidia-docker, but yes it doesn't necessarily seem very hard to make a Docker image for Accelerate.

How would you imagine to develop with it? Would you shell into a running container and do your development there? Or would your build process re-build and re-start the image to run your application? The second way would require some serious amount of caching.

**Comment by tmcdonell** [comment, marked as answer]
Automated docker builds (using nvidia-docker) are located here: https://hub.docker.com/r/tmcdonell/accelerate-llvm (linked from the README)

Please let me know how they work out for you and how you use them for development etc. I've never heard from people using them so they don't get much maintenance time.

**Comment by guissalustiano** [comment]
Wow cool!
That's exactly what I wanted. I search many times in google and docker hub without success.
Thank you!
